applications/app/controllers/CrosswordsController.scala

```diff
@@ -66,10 +67,11 @@ object CrosswordsController extends Controller with ExecutionContexts {
   }
 
   def crosswordsOptOut = Action { implicit request =>
-    Cached(60)(SeeOther("/crosswords?view=old").discardingCookies(
-      DiscardingCookie(
-        CrosswordOptIn,
+    Cached(60)(SeeOther("/crosswords?view=old").withCookies(
```

Comment:
Could we take this opportunity to change the `view` to `classic`? `old` is a bit too harsh!